src/Microsoft.Data.Sqlite.Core/SqliteDataRecord.cs

```diff
@@ -25,20 +25,14 @@ public SqliteDataRecord(sqlite3_stmt stmt)
         public override int FieldCount
             => raw.sqlite3_column_count(_stmt);
 
-        public override double GetDouble(int ordinal)
-            => IsDBNull(ordinal)
-                ? throw new InvalidCastException()
-                : raw.sqlite3_column_double(_stmt, ordinal);
-
-        public override long GetInt64(int ordinal)
-            => IsDBNull(ordinal)
-                ? throw new InvalidCastException()
-                : raw.sqlite3_column_int64(_stmt, ordinal);
-
-        public override string GetString(int ordinal)
-            => IsDBNull(ordinal)
-                ? throw new InvalidCastException()
-                : raw.sqlite3_column_text(_stmt, ordinal);
+        protected override double GetDoubleCore(int ordinal)
```

Comment:
Isn't this a breaking change? Or is it only used by code new in 2.0?

Reply:
New code (and real `internal`)